package.json

```diff
@@ -77,11 +77,17 @@
       "@salesforce/plugin-command-reference"
     ],
     "topics": {
-      "deploy": {
-        "description": "Commands to deploy artifacts to an environment.",
+      "project": {
+        "external": true,
+        "description": "TODO",
```

Comment:
I believe we don't have a clear definition of project yet so I'd suggest to remove the description for now, we can add it latter.